README.md

```diff
@@ -559,6 +560,252 @@ The model can also be downloaded and served locally.
 
    The output from the command will consist of a series of outputs from the model before and after training.
 
+### 🧪 Evaluate the newly trained model
+
+You can use the `ilab` command to evaluate the models you are training with several benchmarks. Currently, four benchmarks are supported.
+
+| Benchmark | Measures | Full Name | Description |
+| --- | --- | --- | --- |
+| MMLU | Knowledge | Massive Multitask Language Understanding | TODO |
+| MMLUBranch | Knowledge | N/A | TODO |
+| MTBench | Skills | Multi-turn Benchmark | TODO |
```

Comment:
The MTBench benchmark tests a model against a standardized set of skills and produces an improvement or regressions score based on this evaluation.

Reply:
I changed this one a good amount and added a reference